src/subdomains/core/payment-link/services/payment-link.service.ts

```diff
@@ -286,6 +286,11 @@ export class PaymentLinkService {
       transferAmounts: actualQuote.transferAmountsForPayRequest,
     };
 
+    await Util.timeout(
+      this.paymentLinkPaymentService.processExpiredPayments(),
+      Util.secondsDiff(actualQuote.expiryDate) * 1000,
+    );
```

Comment:
I think it doesn't make sense to wait for this timeout when creating a payment request.

Reply:
We should start the timer when creating a payment, not for every payment request.

Reply:
`Util.secondsDiff(actualQuote.expiryDate)` will give a negative number?